ansible/playbooks/roles/node/templates/harmony.service.j2

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=harmony service
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+User={{ USER.user }}
+WorkingDirectory={{ USER.home }}
+{% if 'exp' in inventory %}
+ExecStart={{ USER.home }}/node.sh -N {{ network }} -p {{ USER.home }}/.hmy/blskeys/bls.pass -T explorer -i 0 -A -S -M -D -P -1
+{% else %}
+{% if network in ['staking'] %}
+ExecStart={{ USER.home }}/node.sh -z -N {{ network }} -p {{ USER.home }}/.hmy/blskeys/bls.pass -S -M -D -P -1
+{% else %}
+ExecStart={{ USER.home }}/node.sh -N {{ network }} -p {{ USER.home }}/.hmy/blskeys/bls.pass -S -M -D -P -1
+{% endif %}
+{% endif %}
+
```

Comment:
add standarderr etc to this template, check this one, https://github.com/harmony-one/experiment-deploy/blob/master/tools/harmony.service.template

Reply:
already added